src/components/presentations/UserArticles/author-articles.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../../assets/stylesheets/index.scss';
+h3, p {
+    font-family: $font-stack;
+}
+.profile-article-list {
+    text-align: left;
+    width: 47%;
+    margin: 10px;
+    padding: 10px 20px;
+    float: left;
+    overflow: auto;
+    border: solid 1px $article-border-color;
+    border-radius: 4px;
+    box-shadow: 0px 3px #888888;
+    
+    .article-list {
+        width: 100%;
+        float: left;
+        h3 {
+            font-size: 18px;
+        }
+        p {
+            font-size: 16px;
+            color: #868686;
+        }
+    }
+    .update-buttons {
+        width: 100%;
+        margin: 10px auto;
+        float: left;
+
+        .success {
+            background: #fff;
+            color: #20A100;
+            border: solid 1px #20A100;
+        }
+
+        .success:hover {
+            background: #20A100;
+            color: #fff;
+            border: solid 1px #20A100;
+        }
+
+        .failure {
+            background: #fff;
+            color: #D90000;
+            border: solid 1px #D90000;
+        }
+
+        .failure:hover {
+            background: #D90000;
+            color: #fff;
+            border: solid 1px #D90000;
+        }
+
+        .unpublish {
+            background: #857E7E;
+            color: #fff;
+            text-transform: capitalize;
+            border: solid 1px #857E7E;
+        }
+
+        .unpublish:hover {
+            background: #fff;
+            color: #857E7E;
+            text-transform: capitalize;
+            border: solid 1px #857E7E;
+        }
+
+        .publish {
+            background: #fff;
+            color: $primary-color;
+            text-transform: capitalize;
+            border: solid 1px $primary-color;
+        }
+
+        .publish:hover {
+            background: $primary-color;
+            color: #fff;
+            text-transform: capitalize;
+            border: solid 1px $primary-color;
+        }
+
+    } 
+}
```

Comment:
Please fix this EOL.

Reply:
Thanks for the feedback, this has been corrected.